app/views/authors/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<h1><%= @author.name %></h1>
+<div id="book-list">
+  <h2>Books:</h2>
+  <% if @author.book_count == 0 %>
+    <p>This author has no books in the database.</p>
+  <% end %>
+  <% @author.books.each do |book|%>
+    <div id="book-<%= book.id %>">
+      <img src="<%= book.thumbnail %>" alt="Cover of <%= book.title %>">
+      <h3><a href="<%= book_path(book) %>"><%= book.title %></a></h3>
+      <% if book.author_count > 1 %>
+        <p>
+          Co-authored by
+          <% book.coauthors(@author).each_with_index do |coauthor, i| %>
+            <a href="<%= author_path(coauthor) %>"><%= coauthor.name %></a><%= ", " if i < book.author_count - 2 %>
```

Comment:
Possible refactor to controller to pass an argument to the view instead of each with index. Unsure of MVC conventions here.